### ISSUE TITLE
[PY3] TestDumpMaster.test_basic: Un-acked message <Mock>

### ISSUE THREAD

**Issue by mhils** [mitmproxy/mitmproxy#1398]
##### Steps to reproduce the problem:

1. (python3) py.test -k TestDumpMaster -v

##### What is the expected behavior?
No console output.

##### What went wrong?
```
test/mitmproxy/test_dump.py::TestDumpMaster::test_basic Exception ignored in: <bound method Reply.__del__ of <mitmproxy.controller.Reply object at 0x04E10570>>
Traceback (most recent call last):
  File "C:\Users\user\git\mitmproxy\mitmproxy\controller.py", line 267, in __del__
    raise exceptions.ControlException("Un-acked message: %s" % self.obj)
mitmproxy.exceptions.ControlException: Un-acked message: <Mock id='81855280'>
PASSED
```

---

Mitmproxy Version: master
Operating System: Windows x64

**Comment by Kriechi** [comment]
I see these on OSX and even on Travis from time to time.

**Comment by mhils** [comment]
fixed by #1474
